_docker/drupal/Dockerfile

```diff
@@ -44,24 +44,29 @@ RUN curl https://getcomposer.org/installer > composer-installer && php composer-
 
 WORKDIR /var/www/drupal
 COPY apache/drupal.conf /etc/httpd/conf.d/drupal.conf
-COPY scripts /var/www/drupal/scripts
-RUN chmod -v +x /var/www/drupal/scripts/run-httpd.sh
 
-# Get Drupal in here
-# Doing this in multiple steps so less things change on a day to day build and builds take less time
+# Run composer to install Drupal and required dependencies
+COPY drupal-filesystem/scripts/ScriptHandler.php /var/www/drupal/scripts/
 COPY drupal-filesystem/composer.json drupal-filesystem/composer.lock drupal-filesystem/2663638-2.patch drupal-filesystem/view_render_patch.patch /var/www/drupal/
 RUN /usr/local/bin/composer install -n
+ENV PATH=$PATH:/var/www/drupal/vendor/bin
+
+# Set working directory and CMD
+WORKDIR /var/www/drupal/web
+CMD ["/var/www/drupal/run-httpd.sh"]
```

Comment:
CMD sets the command that will be run when the container starts, but at image build time it doesn't run anything (RUN actually runs things and commits them to the layer) so it can be put wherever you want within the Dockerfile.